docs/CLA.md

```diff
@@ -0,0 +1,79 @@
+# RespiraWorks Contributor License Agreement
+
+Thank you for your interest in contributing to RespiraWorks ("We" or "Us").
+
+This contributor agreement ("Agreement") documents the rights granted by contributors to Us. To make this document effective, please sign it and send it to Us by electronic submission, following the instructions at https://cla-assistant.io/. This is a legally binding document, so please read it carefully before agreeing to it. The Agreement may cover more than one software project managed by Us.
+
+## 1. Definitions
+
+"You" means the individual who Submits a Contribution to Us.
+
+"Contribution" means any work of authorship that is Submitted by You to Us in which You own or assert ownership of the Copyright. If You do not own the Copyright in the entire work of authorship, please follow the instructions in https://cla-assistant.io/.
```

Comment:
The link to https://cla-assistant.io/ is a placeholder. This should be updated to reference an page on our site that provides instructions.

Reply:
Fixed to reference the new "Contributing" page.